index.html

```diff
@@ -571,18 +571,13 @@
       <p>
         The <a>PushManager</a> interface defines the operations to access <a>push services</a>.
       </p>
-      <dl title="interface PushManager" class="idl">
-        <dt>
-          Promise&lt;PushSubscription&gt; subscribe (optional PushSubscriptionOptionsInit options)
-        </dt>
-        <dt>
-          Promise&lt;PushSubscription?&gt; getSubscription ()
-        </dt>
-        <dt>
-          Promise&lt;PushPermissionState&gt; permissionState (optional PushSubscriptionOptionsInit
-          options)
-        </dt>
-      </dl>
+      <pre class="idl">
+interface PushManager {
+    Promise&lt;PushSubscription&gt; subscribe(optional PushSubscriptionOptionsInit options);
+    Promise&lt;PushSubscription?&gt; getSubscription();
+    Promise&lt;PushPermissionState&gt; permissionState(optional PushSubscriptionOptionsInit options);
+};
+</pre>
       <p>
         The <code><dfn id=
         "widl-PushManager-subscribe-Promise-PushSubscription--PushSubscriptionOptionsInit-options">subscribe</dfn></code>
```

Comment:
OK, let's do it.

Thanks for your help, Marcos! I'll make sure that I read up on the ReSpec WebIDL guide.